images.yaml

```diff
@@ -197,6 +197,14 @@
   tags:
   - sha: 7d4d712a2686da99d37580b4e2f45eb658b74e4b01caf67c1099adc294b96b52
     tag: v0.6.0
+- name: kong
+  tags:
+  - sha: df7bb41be97ec490d08e7fb5acb2de626042a1eedb16405bc8504814a67140c0
+    tag: 1.2.2
+- name: kong-docker-kubernetes-ingress-controller.bintray.io/kong-ingress-controller
```

Comment:
This is a public jFrog registry. I'm not sure why it's not on Docker Hub.

Reply:
OK it's because the enterprise version is there.

kong-docker-kong-enterprise-edition-docker.bintray.io/kong-enterprise-edition